lab_1_keywords_tfidf/main.py

```diff
@@ -48,7 +74,14 @@ def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(tokens, list) and tokens != []:
```

Comment:
revert the condition and return None immediately

Reply:
done